docs/design/ovs-pipeline.md

```diff
@@ -1,5 +1,57 @@
 # Antrea OVS Pipeline
 
+## Table of Contents
+
+<!-- toc -->
```

Comment:
You should remove this doc from the list in `hack/.notableofcontents`. Then our CI can help ensure that the ToC stays up-to-date.

Reply:
In the current version, I do not add all levels of sub-sections to the ToC. If I remove  the doc from hack/.notableofcontents, then I have to add all sections. Let me know what you think.

Reply:
Got it, sounds good to me